### PR TITLE
Adding clustering 5 models inside single_model_dict #32

### PR DESCRIPTION
**Pull request by Tihsrah**
PR tagged with #32 

I have added 5 clustering models in automl -> models.py .
Name of the clustering models are as follows:
AffinityPropagation
AgglomerativeClustering
Birch
DBSCAN
KMeans
.
DESCRIPTION
AffinityPropagation - It involves finding a set of exemplars that best summarize the data. It takes as input measures of similarity between pairs of data points. Real-valued messages are exchanged between data points until a high-quality set of exemplars and corresponding clusters gradually emerges
.
AgglomerativeClustering- It involves merging examples until the desired number of clusters is achieved.It is implemented via the class AgglomerativeClustering and the main configuration to tune is the “n_clusters” set, an estimate of the number of clusters in the data, e.g. 2.
.
BIRCH -BIRCH Clustering involves constructing a tree structure from which cluster centroids are extracted.
main configuration to tune is the “threshold” and “n_clusters” hyperparameters, the latter of which provides an estimate of the number of clusters.
.
DBSCAN Clustering  involves finding high-density areas in the domain and expanding those areas of the feature space around them as clusters.
the main configuration to tune is the “eps” and “min_samples” hyperparameters.
.
KMEANS -the main configuration to tune is the “n_clusters” hyperparameter set to the estimated number of clusters in the data.

### CODE REVIEW

**Comment by sagnik1511** [comment]
Update the description properly so that you'll get points.

**Comment by Tihsrah** [comment]
done

File: tab_automl/automl/models.py [hunk 1]
```diff
@@ -26,5 +30,12 @@
         "Random Forest Classifier": RandomForestClassifier,
         "XGBoost Classifier": XGBClassifier,
         "KNN Classifier": KNeighborsClassifier
+    },
+    "clustering":{
```
Comment (sagnik1511) [inline tab_automl/automl/models.py]: add proper spacing between the keys and the values of the `clustering` dict.

Comment (Tihsrah) [reply tab_automl/automl/models.py]: done

**Comment by sagnik1511** [comment]
Thanks! Merged

**Pull request merged**